app/views/records/show.html.erb

```diff
@@ -83,6 +83,7 @@
       <% end %>
       </ul>
     </div>
-    
+    <% patient_resource = @record.patient.entry.find { |e| e.resource.resourceType == 'Patient' }.resource %>
+    <%= react_component("PatientView", { data: patient_resource, patient: patient_resource, entry: @relevant_entries, errors: @record.entry_id_validation_hash }) %>
```

Comment:
Looks really good.  Looks like what I did in the Error report has made the error displays a little wonky.

Really Gross Screenshot
![Screen Shot 2021-04-21 at 3 20 33 PM](https://user-images.githubusercontent.com/8173551/115609697-dd450380-a2b5-11eb-9a11-81e62167da19.png)

We can probably strip out the 'entry' and 'resource' elements from the entry_id_validation_hash.  This code works, but you might have a better idea.

`<%= react_component("PatientView", { data: patient_resource, patient: patient_resource, entry: @relevant_entries, errors: @record.entry_id_validation_hash.map { |vh| [vh[0], vh[1].slice('errors', 'warnings', 'information')] }.to_h }) %>`

This makes the error pop-up look like this.

![Screen Shot 2021-04-21 at 3 20 57 PM](https://user-images.githubusercontent.com/8173551/115609649-cef6e780-a2b5-11eb-9ee0-7d3e0f3c9cee.png)